### DISCUSSION TITLE
A suggestion for this version of Otaku.

### DISCUSSION THREAD

**Discussion by Hemicrusher** [Goldenfreddy0703/Otaku-Testing#15] [Ideas]
In the Otaku issues I was told that this version will not replace Otaku because of a few users who are still on older versions of Kodi. How about calling it something like Otaku-San or Otaku-MAL and making it a separate addon, along with Otaku and Otaku-Testing? 

Since Anilist can't seem to get their API working, which according to their Discord they have zero idea what the problem even is, it would be nice to have an addon free of Anilist's API that doesn't say "Testing" in the title.

BTW, do you guys have a way that users can send you a few dollars as thanks for all the work you do?

Thanks!!!

**Comment by Goldenfreddy0703** [comment]
> Otaku-San

Im not gonna lie, i like that name. 😂

Our Testing version of Otaku supports both Anilist and Mal but the user can decide on what kinda of database to use.  Otaku-Mal would not be a good name cause we support both but the other name does sounds tempting so maybe. 

As for donations, we don't really except donations but we may set one up in the future. We mostly work on Otaku cause we love anime and a good addon that can give us both quality and function.

**Comment by Hemicrusher** [comment]
> > Otaku-San
> 
> Im not gonna lie, i like that name. 😂
> 
> Our Testing version of Otaku supports both Anilist and Mal but the user can decide on what kinda of database to use. Otaku-Mal would not be a good name cause we support both but the other name does sounds tempting so maybe.
> 
> As for donations, we don't really except donations but we may set one up in the future. We mostly work on Otaku cause we love anime and a good addon that can give us both quality and function.

Cool!!

Well, hope you have a great weekend!!!

**Comment by jewbmx** [comment]
Your addon situation is why mines got v2 in its title lol.

**Comment by Goldenfreddy0703** [comment]
Ooo v2 in the title may sound better then testing, maybe I will rename it to Otaku v2 instead on the next release.

**Comment by Hemicrusher** [comment]
I got a new one....

Shin Otaku

I think it means, New Otaku
